allensdk/mouse_connectivity/grid/utilities/image_utilities.py

```diff
@@ -248,22 +258,28 @@ def __read_segmentation_image_with_kakadu(path):
         raise OSError('file not found at {}'.format(path))
     return jpeg_twok.read(path).T
 
+
 def __read_intensity_image_with_kakadu(path, reduce_level, channel):
     if not os.path.exists(path):
         raise OSError('file not found at {}'.format(path))
     return jpeg_twok.read(path, reduce_level, channel).T
 
+
 def __read_segmentation_image_with_glymur(path):
     return glymur.Jp2k(path)[:]
 
-def __read_intensity_image_with_glymur():
-    image = glymur.Jp2k(path)[:]
+
+def __read_intensity_image_with_glymur(path):
+    return glymur.Jp2k(path)[:]
```

Comment:
@djkapner I changed this method because it appeared totally broken to the point that the linter was complaining...

Reply:
yeah, hard to see how that ever worked.